dom.bs

```diff
@@ -3040,6 +3040,34 @@ standards that want to define APIs shared between <a for=/>documents</a> and
  <a for=CharacterData>data</a> is the string and <a for=Node>node document</a> is
  <var>document</var>.
 
+ <li><p>Let <var>isScriptElement</var> be <var>parent</var> is a {{HTMLScriptElement}}.
```

Comment:
This reads wonky. I think it needs to be something like:

> Let x be true if y; otherwise false.

Also, it's "an" HTMLScriptElement and that should probably read "an HTMLScriptElement node" or perhaps "a script element". (I forgot if the DOM Standard has precedent for this.)